src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedString = sourceString.trim();
+
+  const declarations = cleanedString.split(';');
+
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. A more descriptive name like `styleObject` could be used instead.